test/bancor-portal/BancorPortal.ts

```diff
@@ -740,4 +846,20 @@ describe('BancorPortal', () => {
     const isBNT = (token: TokenWithAddress): boolean => {
         return token.address === bnt.address;
     };
+
+    const transfer = async (
+        sourceAccount: SignerWithAddress,
+        token: TokenWithAddress,
+        target: string | Addressable,
+        amount: BigNumberish
+    ) => {
+        const targetAddress = toAddress(target);
+        const tokenAddress = token.address;
+        if ([NATIVE_TOKEN_ADDRESS, WETH_ADDRESS].includes(tokenAddress)) {
+            return await sourceAccount.sendTransaction({ to: targetAddress, value: amount });
```

Comment:
We usually don't `return await`- just `return` the promise (it's unneeded and node.js knows how to handle it)

Reply:
done